File: C/INICIANTE/2003 Domingo de Manhã.c
#include <stdio.h>

int main()
{
    int horas, minutos;
    char pontos;
    while (scanf("%d %c %d", &horas, &pontos, &minutos) != EOF)
    {
        int atraso = 0;
        if (((horas <= 7) && (minutos == 0)) || (horas < 7))
        {
            printf("Atraso maximo: 0\n");
        }
        else
        {
            if (horas == 9)
                printf("Atraso maximo: %d\n", (60*2));
            else if (horas == 8)
                printf("Atraso maximo: %d\n", (60+minutos));    
            else
                printf("Atraso maximo: %d\n", (minutos));
        }
    }
    return 0;
}

File: Python/INICIANTE/2003 Domingo de Manhã.py
# SUBPROGRAMA
# PROGRAMA PRINCIPAL
while True:
    try:
        array = list(input().split())
        horas = int(array[0][0])
        minutos = int((array[0][2] + array[0][3]))
        minutos = int(minutos)
        if ((horas <= 7) and (minutos == 0)) or (horas < 7):
            print("Atraso maximo: 0")
        else:
            if horas == 9:
                print("Atraso maximo: %d" % (60*2))
            elif horas == 8:
                print("Atraso maximo: %d" % (60+minutos))
            else:
                print("Atraso maximo: %d" % minutos)
    except EOFError:
        break
